services/visual_recognition/v3.js

```diff
@@ -309,37 +300,42 @@ module.exports = function (RED) {
           async.parallel(asyncTasks, function(error, results){
             // when all temp local copies are ready, 
             // copy of all parameters and request to watson api
-            console.log(error, results);
+<<<<<<< HEAD
+            //console.log(error, results);
             if (error)
             {
-              console.log('Parallel ended with error ' + error);
+              console.log('createClassifier ended with error ' + error);
               throw error;
             }
             for (p in listParams)
               params[p]=listParams[p];
-            performAction(params, feature, actionComplete2);
+            performAction(visualRecognition, params, feature, actionComplete2);
           });
       }
-        else if (feature === 'retrieveClassifiersList') { 
-          performAction(params, feature, actionComplete2);
+        else if (feature==='retrieveClassifiersList') { 
+          performAction(visualRecognition, params, feature, actionComplete2);
       }
-        else if (feature === 'retrieveClassifierDetails') {
+        else if (feature ==='retrieveClassifierDetails') {
           params['classifier_id']=msg.params['classifier_id'];
-          performAction(params, feature, actionComplete2);
+          performAction(visualRecognition, params, feature, actionComplete2);
       }
-        else if (feature === 'deleteClassifier') {
+        else if (feature=='deleteClassifier') {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2712969112/issues/source?bid=3330694&fileBranchId=3344618#l322)